Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerFragment.kt

```diff
@@ -198,6 +203,19 @@ class QrCodeScannerFragment : Fragment(R.layout.fragment_qrcode_scanner), AutoIn
         .setOnDismissListener { startDecode() }
         .show()
 
+    private fun showValidationServiceMinVersionDialog(errorMsg: LazyString) {
+        val msg = errorMsg.get(requireContext())
+        val dialogType = DccTicketingDialogType.ErrorDialog(
+            msg = msg,
+            negativeButtonRes = R.string.dcc_ticketing_error_min_version_google_play
+        )
+        dialogType.show(
+            this,
+            positiveButtonAction = { startDecode() },
+            negativeButtonAction = { requireContext().openGooglePlay() }
```

Comment:
I could not find the requirement to redirect. Is it from a chat or so?

Reply:
See https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.14?node-id=40933%3A26462